libs/ledgerjs/packages/types-live/src/operation.ts

```diff
@@ -53,6 +53,7 @@ export type OperationType =
   | "UNSTAKE"
   | "WITHDRAW_UNSTAKED";
 
+export type OperationExtra = object;
```

Comment:
I strongly disagree. If we want to use Typescript for the "type system", we need to define types to know and check the parameters that are used in the code. `unknown` is a gently `any`.

But this PR is far too long so I will stop argue on this point.

Reply:
@haammar-ledger How do I manage the return type of `fromOperationExtraRaw` and `fromOperationExtra`, which are defined in AccountBridge now?

Reply:
I see your point, my suggestion would then be:

```
export type OperationExtra = unknown;
export type Operation<OperationExtra> = {
...
```

wdyt?

Really interested to discuss, but I see `unknown` as much better than `any`, because it forces to make no implicit assumption when handling it.
On the other side `object` doesn't enforce much, I think it's much closer to `any` than `unknown`.